packages/iov-keycontrol/src/wallets/secp256k1hdwallet.spec.ts

```diff
@@ -123,7 +123,7 @@ describe("Secp256k1HdWallet", () => {
     );
 
     const valid = await Secp256k1.verifySignature(
-      ExtendedSecp256k1Signature.fromFixedLength(signatureBytes),
+      ExtendedSecp256k1Signature.fromDer(signatureBytes),
```

Comment:
Also revert this one. Fixed length encoding is the default now, as it contains all the data.